src/GitHub/Data/Webhooks/Payload.hs

```diff
@@ -64,7 +64,7 @@ instance FromJSON OwnerType where
           "organization"  -> pure OwnerOrganization
           _               -> fail $ "Unknown owner type: " ++ T.unpack t
 
-
+-- | Github URL
```

Comment:
Documentation should be valid English sentences that end with a full stop, for example:
```
-- | Triggered when an issue comment is created, edited, or deleted.
-- See <https://developer.github.com/v3/activity/events/types/#issuecommentevent>.
```
Note also that 'Github' should not be used in place of 'GitHub', ie see the copyright at the bottom of this page, '© 2018 GitHub, Inc.'

In this instance
```
-- | Represents a URL that would be suitable to query for more information. The GitHub API only returns valid URLs.
```

Reply:
What is the significance of the statement `The GitHub API only returns valid 'URL's`

Reply:
Good question.

GitHub only returns URLs that are valid, ie they start with https:// or such. If you can imagine a ‘validateUrl :: Text -> Maybe URL’ . That’s the information that is stored by the opaque type URL.

getUrl discards this information.

Reply:
I am not sure where do I see `getUrl` discard an invalid URL? Or maybe I misread your statement? As far as the notation goes, it is only unboxing `URL Text` to `Text`.